inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/Span.java

```diff
@@ -120,7 +120,7 @@ public String toAnnoFileString()
             "page = " + page + "\n" +
             "label = \"" + label + "\"\n" +
```

Comment:
It there also a risk that the label contains special characters that need to be escaped (cf. text)?

Reply:
I think currently not, however definitely worth escaping it nevertheless.
Will fix this.